ethereum/contracts/pyth/Pyth.sol

```diff
@@ -216,6 +216,15 @@ abstract contract Pyth is PythGetters, PythSetters, AbstractPyth {
         }
     }
 
+    function parsePriceFeedUpdates(
+        bytes[] calldata updateData,
+        bytes32[] calldata priceIds,
+        uint64 minPublishTime,
+        uint64 maxPublishTime
+    ) external payable override returns (PythStructs.PriceFeed[] memory priceFeeds) {
+        // TODO: To be implemented soon.
```

Comment:
Might be nice to put a `revert("unimplemented")` here

Reply:
makes sense. thank!